packages/segmented-control/src/SegmentedControlOption.tsx

```diff
@@ -353,23 +94,14 @@ export const SegmentedControlOption = forwardRef<
             aria-controls={ariaControls}
             disabled={disabled}
             className={cx(buttonStyle, {
-              [buttonFocusStyle[mode]]: usingKeyboard,
+              [buttonFocusStyle[theme]]: usingKeyboard,
             })}
             ref={buttonRef}
             onClick={onClick}
             onMouseEnter={onMouseEnter}
             onMouseLeave={onMouseLeave}
           >
-            <span
-              className={cx(labelStyle, {
-                // TODO: Refresh - remove darkmode font override
-                [css`
-                  font-family: ${fontFamilies.legacy};
-                `]: mode === 'dark',
-              })}
-            >
-              {children}
-            </span>
+            <span className={labelStyle}>{children}</span>
```

Comment:
Can we add another `span` here so consumers don't have to?

Reply:
Oh I see, this affects icons being centered... I still think we should try and find a solution that doesn't require extra work on the consumers' part. Not sure atm what that should be

Reply:
Does this help at all? https://css-tricks.com/flexbox-truncated-text/